Tribler/Test/Community/channel/test_channel_base.py

```diff
@@ -0,0 +1,20 @@
+from Tribler.Test.test_as_server import AbstractServer
+from Tribler.community.channel.community import ChannelCommunity
+from Tribler.dispersy.dispersy import Dispersy
+from Tribler.dispersy.endpoint import ManualEnpoint
+from Tribler.dispersy.member import DummyMember
+from Tribler.dispersy.util import blocking_call_on_reactor_thread
+
+
+class AbstractChannelCommunity(AbstractServer):
```

Comment:
Please prefix this with `Test` since the name might suspect that it's part of the Core.